pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/rest/api/v2/SinkApiV2Resource.java

```diff
@@ -133,42 +130,42 @@ public Response getSinkInfo(final @PathParam("tenant") String tenant,
     @Produces(MediaType.APPLICATION_JSON)
     @Path("/{tenant}/{namespace}/{sinkName}/status")
     public SinkStatus getSinkStatus(final @PathParam("tenant") String tenant,
-                                  final @PathParam("namespace") String namespace,
-                                  final @PathParam("sinkName") String sinkName) throws IOException {
+                                    final @PathParam("namespace") String namespace,
+                                    final @PathParam("sinkName") String sinkName) throws IOException {
         return sink.getSinkStatus(tenant, namespace, sinkName, uri.getRequestUri());
     }
 
     @GET
     @Path("/{tenant}/{namespace}")
-    public Response listSink(final @PathParam("tenant") String tenant,
-                             final @PathParam("namespace") String namespace) {
+    public List<String> listSink(final @PathParam("tenant") String tenant,
```

Comment:
this is returning `List<String>` instead of `Response`. doesn't it break the BC?